packages/pcre2.rb

```diff
@@ -1,44 +1,60 @@
+  source_sha256 'ded42661cab30ada2e72ebff9e725e745b4b16ce831993635136f2ef86177724'
 
   binary_url({
-    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/pcre2/10.36_armv7l/pcre2-10.36-chromeos-armv7l.tar.xz',
-     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/pcre2/10.36_armv7l/pcre2-10.36-chromeos-armv7l.tar.xz',
-       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/pcre2/10.36_i686/pcre2-10.36-chromeos-i686.tar.xz',
-     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/pcre2/10.36_x86_64/pcre2-10.36-chromeos-x86_64.tar.xz'
+    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/pcre2/10.40_armv7l/pcre2-10.40-chromeos-armv7l.tar.zst',
+     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/pcre2/10.40_armv7l/pcre2-10.40-chromeos-armv7l.tar.zst',
+       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/pcre2/10.40_i686/pcre2-10.40-chromeos-i686.tar.zst',
+     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/pcre2/10.40_x86_64/pcre2-10.40-chromeos-x86_64.tar.zst'
   })
   binary_sha256({
-    aarch64: 'f83694fcc265d5dc06c627e47ea7765e74eb337727277f4dbbd8df7d1d5e8114',
-     armv7l: 'f83694fcc265d5dc06c627e47ea7765e74eb337727277f4dbbd8df7d1d5e8114',
-       i686: '845f8a7fa844ea982be1b6a81e5f48d23be348861f6d33fd84ca5b98dd032b54',
-     x86_64: '17002f3ed5ebba3944dc1050a314e75b03a25a0bfc54ac1c12d1060178456199'
+    aarch64: 'e81850b864c3af34d3ebf7a24620d3c9c612a89fc75a92f2191d83aad80ec487',
+     armv7l: 'e81850b864c3af34d3ebf7a24620d3c9c612a89fc75a92f2191d83aad80ec487',
+       i686: 'e8416aebe3c3b14b1523a3a1b57a5bf947e700517714ab96db406edc0c1af0dd',
+     x86_64: '1e09d4a30ca30ad9f4103d6fe5ba865cf87a0be83c97f3e8d4e0a543736f7b6c'
   })
 
   depends_on 'glibc' # R
   depends_on 'readline' # R
+  depends_on 'bz2' # R
+  depends_on 'zlibpkg' # R
 
   def self.build
-    system "./configure #{CREW_OPTIONS} \
-      --enable-shared \
-      --with-pic \
-      --enable-unicode-properties \
-      --enable-pcre2-16 \
-      --enable-pcre2-32 \
-      --enable-jit \
-      --enable-pcre2test-libreadline"
-    system 'make'
+    FileUtils.mkdir_p 'builddir'
+    Dir.chdir 'builddir' do
+      system "env cmake \
+        -G Ninja \
+        #{CREW_CMAKE_OPTIONS} \
+        -DPCRE2_BUILD_TESTS=OFF \
+        -DBUILD_SHARED_LIBS=ON \
+        -DPCRE2_SUPPORT_JIT=ON \
+        -DPCRE2_STATIC_PIC=ON \
+        -DPCRE2_BUILD_PCRE2_16=ON \
+        -DPCRE2_BUILD_PCRE2_32=ON \
+        -Wno-dev \
+        .."
+    end
+    system 'ninja -C builddir'
   end
 
   def self.install
-    system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
+    system "DESTDIR=#{CREW_DEST_DIR} ninja -C builddir install"
+    FileUtils.mkdir_p CREW_DEST_MAN_PREFIX
+    FileUtils.mkdir_p "#{CREW_DEST_LIB_PREFIX}/cmake/pcre2"
+    FileUtils.mv Dir.glob("#{CREW_DEST_PREFIX}/man/*"), "#{CREW_DEST_MAN_PREFIX}/"
```

Comment:
I'm not sure what this does.  Isn't `#{CREW_DEST_PREFIX}/man` the same as `#{CREW_DEST_MAN_PREFIX}`?

Reply:
Nope.

```
chronos@cheekon-i686 /usr/local/lib/crew/packages $ crew const CREW_DEST_MAN_PREFIX
CREW_DEST_MAN_PREFIX=/usr/local/tmp/crew/dest/usr/local/share/man
```

Reply:
The installer is putting files in /use/local/man/man[1,3], and those folders need to be moved to handle the FHS check.

Reply:
Yes, you are correct.